src/SwaggerProvider/SwaggerProvider.fsproj

```diff
@@ -10,10 +10,11 @@
     <RootNamespace>SwaggerProvider</RootNamespace>
     <AssemblyName>SwaggerProvider</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <TargetFSharpCoreVersion>4.4.0.0</TargetFSharpCoreVersion>
+    <TargetFrameworkProfile>Profile259</TargetFrameworkProfile>
+    <TargetProfile>netcore</TargetProfile>
+    <TargetFSharpCoreVersion>3.1.2.5</TargetFSharpCoreVersion>
```

Comment:
Definitly `Profile259`,  I've created PCL project in VS 2017 and manually merge changes in existing SwaggerProvider.fsproj file, probably did something wrong (this is my first PCL proj)

This is what I get from VS2017
```xml
  <PropertyGroup>
    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
    <SchemaVersion>2.0</SchemaVersion>
    <ProjectGuid>0d200fdc-8f8a-43a0-bd11-d30b53eb98f8</ProjectGuid>
    <OutputType>Library</OutputType>
    <RootNamespace>PortableLibrary_Profile259</RootNamespace>
    <AssemblyName>PortableLibrary_Profile259</AssemblyName>
    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
    <TargetFrameworkProfile>Profile259</TargetFrameworkProfile>
    <TargetProfile>netcore</TargetProfile>
    <TargetFSharpCoreVersion>3.259.41.0</TargetFSharpCoreVersion>
    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
    <Name>PortableLibrary_Profile259</Name>
  </PropertyGroup>
```